backend/satellite_tools/repo_plugins/deb_src.py

```diff
@@ -62,7 +62,7 @@ def is_populated(self):
 
 class DebRepo(object):
     # url example - http://ftp.debian.org/debian/dists/jessie/main/binary-amd64/
-    def __init__(self, url, cache_dir, pkg_dir):
+    def __init__(self, url, cache_dir, pkg_dir, proxy_a="", proxy_u="", proxy_p=""):
```

Comment:
Please use more meaningful names for the parameters of this function.

For example: `proxy_addr`, `proxy_user`, and `proxy_pass` are better names here?

Reply:
|I've done the suggested changes